### DISCUSSION TITLE
Error on auth/login after Upgrade to 3.2.20

### DISCUSSION THREAD

**Discussion by pushpanda** [statamic/cms#4641] [Troubleshooting] [answered]
After upgrading to the lastest version 3.2.20 on production i get an auth error when login into the cp:

`[2021-11-02 21:48:18] production.ERROR: Call to undefined method Statamic\Stache\Query\UserQueryBuilder::intersectKeysFromWhereClause() {"exception":"[object] (Error(code: 0): Call to undefined method Statamic\\Stache\\Query\\UserQueryBuilder::intersectKeysFromWhereClause() at /home/forge/www.foo.com/vendor/statamic/cms/src/Stache/Query/UserQueryBuilder.php:30)`

Statamic Pro 3.2.20, Multi-Site, Forge

on my local dev env everything works smooth.

any ideas?

thx

**Comment by jasonvarga** [comment]
Strange. Do you get that error consistently or was it only once?

Can you try a `composer install` on production?
Or delete the `vendor` directory (or at least `vendor/statamic/cms`) and try `composer install` again?

**Comment by pushpanda** [comment, marked as answer]
Totally strange as it is working locally. I already tried to reinstall the statamic package, didn't work either.

I could now fix the problem with a `git reset`
